### ISSUE TITLE
Mouse doen't have full travel in full screen

### ISSUE THREAD

**Issue by BGPierce** [VCCE/VCC#6]
In working in "Vcc v2.01" on my NitrOS9 C projects, which are almost all GUI (mouse) based, I noticed that when I use "Full Screen" mode, the "coco mouse" does not travel full screen left/right and up/down. The mouse will only travel about 2 thirds of the screen from the left and top edges. It will not go all the way to the far right nor bottom of the screen.
This works fine in "Windowed" mode even when the Vcc window is "Maximized". In Full Screen, the real windows mouse cursor disappears (as it should) leaving only the NitrOS9 mouse cursor, but the cursor only reaches 2/3rds of the full screen. I know this was not a problem in old versions
This is probably a scaling bug and I imagine, easily fixed.

**Comment by BGPierce** [comment]
I just received a build of Gary's VCC 201 and the mouse problem disappears in his version.
We need to merge that version with the 2.100b branch soon as it seems to have fixed a few things. It still doesn't fix the F10 statusbar toggle issue, but does solve the mouse issue. I'm sending a copy to Robert Gault and see if it solves his WinXP "Not a Win 32 application" problem. I'll also sending him the "C++ 2015" runtime as I believe that's his problem.

**Comment by ursine** [comment]
It shouldn't have differed much.  It is a straight build of the vcc branch.  Perhaps my compiler settings differ?

**Comment by BGPierce** [comment]
With the new build that Joe did to fix the WinXP problem, this has gone away again. The Coco mouse now has full travel on Win Vista & WinXP. This seems to be some sort of random thing. So far, it's popped up on about every other build.

**Comment by BGPierce** [comment]
In Joe's last build, this popped up again and the mouse didn't have full travel.
I finally got VS 2015 installed, did a compile, and the problem disappeared again.

**Comment by BGPierce** [comment]
I think I have narrowed this issue down now to a Windows Vista quirk. It may also be the same in other Versions of Windows, but I haven't tested it yet.
It seems that if I set the "properties" of the VCC shortcut to start "Maximized", then when I switch to "Full Screen" mode in VCC (F11), the mouse will only travel 3/4 of the screen in either direction. It has full travel while just a maximized window.
If I set VCC "properties" to start "Normal" (small default window), then switch to "Full Screen" mode, the mouse has full travel.
Since this seems to be on my machine in partitcualr, and no one else is complaining, I am closing this issue as a "Vista Quirk".
